Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/ContextMtl.mm

```diff
@@ -1646,6 +1650,19 @@ GLint GetOwnershipIdentity(const egl::AttributeMap &attribs)
     return angle::Result::Stop;
 }
 
+angle::Result ContextMtl::bindMetalRasterizationRateMap(gl::Context *context,
+                                                        GLMTLRasterizationRateMapANGLE map)
+{
+    id<MTLRasterizationRateMap> rateMap = (__bridge id<MTLRasterizationRateMap>)(map);
+    if (rateMap && rateMap.device != mContextDevice.get())
+    {
+        return angle::Result::Stop;
```

Comment:
Does this cause an OpenGL ES error to be signaled to the application? If not, should it?

Reply:
I've made a note that this needs a test.